spec/controllers/users_controller_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'rails_helper'
 RSpec.describe UsersController, type: :controller do
-  let!(:user1) { FactoryBot.create(:user) }
+  let(:user) { FactoryBot.create(:user) }
+  before do
+    stub_current_user(user)
+    stub_authorize
+  end
```

Comment:
We can now call this method here:
https://github.com/mla-wtag/ml_Journey/pull/6/files#r1349867680